src/test/java/io/knotx/launcher/KnotxStarterVerticleTest.java

```diff
@@ -31,8 +32,24 @@
 @ExtendWith(VertxExtension.class)
 class KnotxStarterVerticleTest {
 
+  private static final String BOOTSTRAP_TEMPLATE = "simple/bootstrap.json";
   private static final String MY_VALUE_KEY = "myValueKey";
 
+  @Test
+  @DisplayName("Unsupported Knot.x 1.0 format validation")
+  void startWithInvalidModules(VertxTestContext testContext, Vertx vertx) {
+    // given
+    DeploymentOptions options = fromBootstrapTemplate(BOOTSTRAP_TEMPLATE, "invalid.conf");
+
+    // when
+    vertx.rxDeployVerticle(KnotxStarterVerticle.class.getName(), options)
+        .subscribe(
+            // then
+            success -> testContext.failNow(new RuntimeException("This deployment should fail")),
+            error -> testContext.completeNow()
```

Comment:
Can we check here that this is particular exception or at least its message?
So that we are sure what caused the error, not it just failed.